static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.ts

```diff
@@ -1,14 +1,19 @@
 import config from '../../../../../lib/config';
-import { getHeaderBiddingAdSlots } from '../slot-config';
+import type { Advert } from '../../dfp/Advert';
 import { dfpEnv } from '../../dfp/dfp-env';
+import { getHeaderBiddingAdSlots } from '../slot-config';
 
 /*
  * Amazon's header bidding javascript library
  * https://ams.amazon.com/webpublisher/uam/docs/web-integration-documentation/integration-guide/javascript-guide/display.html
  */
 
-class A9AdUnit {
-	constructor(advert, slot) {
+class A9AdUnit implements A9AdUnitInterface {
+	slotID?: string;
+	slotName?: string;
+	sizes?: HeaderBiddingSize[];
+
+	constructor(advert: Advert, slot: HeaderBiddingSlot) {
 		this.slotID = advert.id;
 		this.slotName = config.get('page.adUnit');
```

Comment:
Could we set a fallback, here?

```suggestion
		this.slotName = config.get<string>('page.adUnit','unknown-ad-unit');
```

Reply:
We could do but would we be doing this just so that `slotName` could be of type `string` rather than `string | undefined`? It seems to me that `undefined` is a better value to indicate that adUnit is unknown.